builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -344,10 +344,38 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/stg-config.json"
 			sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/prod-config.json"
 
-      sh "sed -i -- 's/{conf_deployment_targets}/${config_loader.JAZZ.DEPLOYMENT_TARGETS}/g' ./config/dev-config.json"
-      sh "sed -i -- 's/{conf_deployment_targets}/${config_loader.JAZZ.DEPLOYMENT_TARGETS}/g' ./config/stg-config.json"
-      sh "sed -i -- 's/{conf_deployment_targets}/${config_loader.JAZZ.DEPLOYMENT_TARGETS}/g' ./config/prod-config.json"
-      sh "sed -i -- 's/{conf_deployment_targets}/${config_loader.JAZZ.DEPLOYMENT_TARGETS}/g' ./config/test-config.json"
+      def apiOptions=""
+      def functionOptions=""
+      def websiteOptions=""
+      for (String item: config_loader.JAZZ.DEPLOYMENT_TARGETS.API) {
```

Comment:
had to do this as we want the array/list of options to be added and sed -i 's/ does only string replacement and we also want to retain the quotes. If there is any better way to do it I am all ears